### PR TITLE
Sending PR title with version to slack and update to readme.md

### PR DESCRIPTION
**Pull request by jo-bisonai**
PR for Continuous Integration/Continuous Deployment (CICD) Improvement of 'orakl'

Tested Applications: "por", "cli", "vrf", "sentinel"

Main Features:

1. Ability to select image versions during dispatch. 
2. Enhancement: Updating image versions directly in dispatch (previously only via helm-chart).
3. Cypress integration: Updating images for Cypress tests based on selected versions.
4. Slack integration: Posting PR titles with associated image versions.
5/ Documentation update: README.md updated to include image tags.

Note: If the list of images becomes too extensive, periodic cleaning may be necessary.

### CODE REVIEW

File: .github/workflows/deployment.yaml [hunk 1]
```diff
@@ -19,72 +25,93 @@ on:
...
         required: true
         type: choice
+        default: "new"
         options:
-          - "Baobab"
-          - "Cypress"
-
+          - new
+          - por:v0.0.1.20240621.0703.82429c9:Update price pair tags
+          - vrf:v0.5.1.20240621.0710.dfa5c7d:Update price pair tags
```
Comment (nick-bisonai) [inline .github/workflows/deployment.yaml:36]: are these tags for the testing purpose?

**Comment by nick-bisonai** [comment]
`periodic cleaning may be necessary.` -> do the periodic cleaning stand by removing images from ecr?

File: .github/workflows/deployment.yaml (hunk 1 above)
Comment (jo-bisonai) [reply .github/workflows/deployment.yaml:36]: Yes, it was created for testing. 
we can remove it for fresh new one~

**Comment by jo-bisonai** [comment]
> `periodic cleaning may be necessary.` -> do the periodic cleaning stand for removing images from ecr?

I mean the list will stored in git action file, it it is too many it can cause confusion later. 
Its recommended after some times to clean up un necessary lists from the actions.

**Review by nick-bisonai — APPROVED** [review]
LGTM!

**Pull request merged**